db/toblock.c

```diff
@@ -2558,7 +2558,9 @@ static inline int check_for_node_up(struct ireq *iq, block_state_t *p_blkstate)
 static int toblock_main_int(struct javasp_trans_state *javasp_trans_handle,
                             struct ireq *iq, block_state_t *p_blkstate)
 {
+#ifdef DEBUG_BLKSEQ
```

Comment:
The scattered `#ifdef DEBUG_BLKSEQ` blocks make the code somewhat difficult to read. If the trace is considered too verbose even under the DEBUG verbosity, maybe we should get rid of it?

Reply:
yeah...probably can remove.

Reply:
Left this part as it was before, for now.